compute_tools/src/spec_apply.rs

```diff
@@ -47,7 +47,7 @@ pub enum PerDatabasePhase {
     DeleteDBRoleReferences,
     ChangeSchemaPerms,
     HandleAnonExtension,
-    DropSubscriptionsForDeletedDatabases,
+    DropLogicalSubscriptions,
```

Comment:
I assume this variant is not being used by cplane right now and we won't run into compatibility issues?

Reply:
I'm not sure I got the problem.
This code is internal to compute_ctl.

Reply:
oh, I misunderstood :) LGTM